cinema/serializers.py

```diff
@@ -1,6 +1,21 @@
 from rest_framework import serializers
+from cinema.models import (Genre,
+                           Actor,
+                           CinemaHall,
+                           Movie,
+                           MovieSession,
+                           Order,
+                           Ticket)
 
-from cinema.models import Genre, Actor, CinemaHall, Movie, MovieSession
+
```

Comment:
Ensure there is a blank line at the end of the file to adhere to code style guidelines.